auth-web/src/components/pay/PayWithOnlineBanking.vue

```diff
@@ -47,7 +47,7 @@
         <li>Sign in to your financial institution's online banking website or app</li>
         <li>Go to your financial institution's bill payment page</li>
         <li>Enter "BC Registries and Online Services" as payee</li>
```

Comment:
"BC Registries and Online Services" color seems need to update, check with Ethan.